packages/components/src/molecules/SearchInputField/SearchInputField.tsx

```diff
@@ -89,10 +93,10 @@ const SearchInputField = forwardRef<
       />
       <IconButton
         type="submit"
-        aria-label="Submit Search"
+        aria-label={buttonAriaLabel ?? 'Submit Search'}
         icon={buttonIcon ?? <Icon name="MagnifyingGlass" />}
         size="small"
-        {...buttonProps}
+        {...restButtonProps}
```

Comment:
```suggestion
        {...otherButtonProps}
```

sorry, I forgot to mention here too.